src/test/java/essenmakanan/RecipeParserTest.java

```diff
@@ -0,0 +1,31 @@
+package essenmakanan;
+
+import essenmakanan.exception.EssenMakananException;
+import essenmakanan.parser.RecipeParser;
+import essenmakanan.recipe.RecipeList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RecipeParserTest {
+
+    RecipeParser recipeParser;
+    RecipeList recipes;
+
+    @BeforeEach
+    public void setUp() {
+        recipeParser = new RecipeParser();
+        recipes = new RecipeList();
+    }
+
+    @Test
+    public void parse_invalidCommand_exceptionThrown() {
+        try{
+            recipeParser.parseRecipeCommand(recipes, "invalid command", "invalid details");
```

Comment:
Would it be better if you create 2 variables - invalidCommand and invalidDetails - and give them user like input to pass through the parser?